tests/unit/scripts/test_update_todos.py

```diff
@@ -41,24 +41,23 @@ def test_todo_detection(tmp_path):
 
     file.write_text(
         """
-            # TODO [LOW]: valid todo
+            # TODO [ENG-345]: valid todo
```

Comment:
Oh shit I just detected another issue: This is a test file to test the todo script, which now got modified as it's in the tests directory :D 
We probably have to hard code some exclude for this, and definitely close the created jira issues

Reply:
I just added the option to put a `# notodo` flag in a file, inspired by `noqa`.... just have to write a test for it – I didn't even realize there was a test suite for the script, else I would have adjusted that anyway.

Reply:
Love the `# notodo` tag! Yeah I completely forgot as well that we might have some issues with that file being in there